procrastinate/retry.py

```diff
@@ -17,15 +17,19 @@ class BaseRetryStrategy:
     Children classes only need to implement `get_schedule_in`.
```

Comment:
nit: s/Children/Child (while you're at it)

Reply:
Isn't `children` the proper form as `classes` is plural? :thinking:

Reply:
Ah maybe not, as `child` is an adjective here :) .